demos/microcloud-deploy/templates/mc-init.tmpl

```diff
@@ -0,0 +1,47 @@
+# `lookup_subnet` limits the subnet when looking up systems with mDNS.
+lookup_subnet: ${microcloud_one_address}
+# `lookup_interface` limits the interface when looking up systems with mDNS.
+lookup_interface: ${bridge_nic}
+
+# `systems` lists the systems we expect to find by their host name.
+#   `name` represents the host name
+#   `ovn_uplink_interface` is optional and represents the name of the interface reserved for use with OVN.
+#   `storage` is optional and represents explicit paths to disks for each system.
+systems:
+%{ for mc_inst in instances ~}
+- name: ${mc_inst}
+  ovn_uplink_interface: ${bridge_nic}
+  storage:
+      ceph:
+      - path: /dev/sdb
+        wipe: true
+%{ endfor ~}
+# `ceph` is optional and represents the Ceph global configuration
+ceph:
+  internal_network: ${bridge_nic_cidr}
+  public_network: ${bridge_nic_cidr}
+
+# `ovn` is optional and represents the OVN & uplink network configuration for LXD.
+ovn:
+  ipv4_gateway: ${ovn_gateway}
+  ipv4_range: ${ovn_range_start}-${ovn_range_end}
+  dns_servers: 8.8.8.8
+
+# `storage` is optional and is used as basic filtering logic for finding disks across all systems.
+# Filters are checked in order of appearance.
+# The names and values of each key correspond to the YAML field names for the `api.ResouresStorageDisk`
+# struct here:
+# https://github.com/canonical/lxd/blob/c86603236167a43836c2766647e2fac97d79f899/shared/api/resource.go#L591
+# Supported operands: &&, ||, <, >, <=, >=, ==, !=, !
```

Comment:
To cleanup the template, it would be worth removing the comments from the official docs together with any unsused keys.
If you want to learn more about the keys, you can always consult the official docs.

Maybe having a link to the [docs](https://canonical-microcloud.readthedocs-hosted.com/en/latest/microcloud/how-to/initialise/#non-interactive-configuration) at the beginning of the file would be enough.